Kickstarter-iOS/Views/Controllers/EditorialProjectsViewController.swift

```diff
@@ -68,7 +83,21 @@ public final class EditorialProjectsViewController: UIViewController {
       |> \.backgroundColor .~ UIColor.white
 
     _ = self.headerView
-      |> \.backgroundColor .~ UIColor.hex(0x00007D)
+      |> UIView.lens.layoutMargins %~~ { _, view in
```

Comment:
I added this in to made the paragraph narrower on iPad, but we still need to think about what to do with the image. This approach worked for the cell but with the edge-to-edge header it looks strange. We might just have to make the header _look_ like the cell and give it rounded corners?